src/kirby/components/modal/modal-wrapper/modal-wrapper.component.spec.ts

```diff
@@ -9,7 +9,7 @@ import { IconComponent } from '../../icon/icon.component';
 import { ModalWrapperComponent } from './modal-wrapper.component';
 import { IModalController } from '../services/modal.controller.interface';
 
-describe('ModalWrapperComponent', () => {
+fdescribe('ModalWrapperComponent', () => {
```

Comment:
Oopsie ;)

Reply:
was just thinking that i might have forgotten to remove the fdescribe :D ...